### ISSUE TITLE
some problems

### ISSUE THREAD

**Issue by Drebin54** [AllanWang/Frost-for-Facebook#1924]
- [ ] I have verified that the bug does not occur in my mobile browser (at https://touch.facebook.com/)

**Describe the bug**

Interesting app. However, I encountered some problems. You can't see a response from a response from a comment. We are unable to download the images posted while it works well for video. The search returns us directly to the homepage.

**To Reproduce**
Steps to reproduce the behaviour:

**Details (please provide at least the app version):**
 - App Version: the last
 - Device: Mi10tLight
 - Android Version: 12.1

**Logs/Screenshots:**

**Comment by syphyr** [comment]
> You can't see a response from a response from a comment.

Yes, this seems to be a newly introduced bug in latest version. Interestingly, the responses become visible after commenting in the same thread.

**Comment by Drebin54** [comment]
Alright, hopefully this will all be fixed. For once we come across an interesting FB application to use

**Comment by syphyr** [comment]
I just tested 3.1.2 release and the reply to the reply is still not visible unless I have commented in the thread.  So, this must be caused by some recent change by Facebook and not something that was broken in the 3.2.0 changes.

**Comment by syphyr** [comment]
I just realized that the "reply to reply" is always visible. I was just clicking on the wrong area. You have to click on the persons icon to see it. If you select next to the icon, then it says the message can not be found!

**Comment by Drebin54** [comment]
> I just realized that the "reply to reply" is always visible. I was just clicking on the wrong area. You have to click on the persons icon to see it. If you select next to the icon, then it says the message can not be found!

Indeed it works. I was not clicking on the right area. Thanks a lot for the tip :)

**Comment by Drebin54** [comment]
ADS is back 😔, why?
